tests/unit/swagger/templateHelpers.spec.ts

```diff
@@ -172,34 +172,60 @@ describe('ValidationService', () => {
     });
 
     describe('Enum validate', () => {
+        const name = 'name';
+        type Enumeration = string[] | number[];
+        interface FieldErrors { [key: string]: any; }
+        let validateEnum: (value: any, enumeration: Enumeration, fieldErrors?: FieldErrors) => any;
+
+        beforeEach(() => {
+          const validationService = new ValidationService({});
```

Comment:
Personally I don't really like having logic in the tests. I'd rather have a bit of repetition but the meaning obvious.